samoa-api/src/main/java/org/apache/samoa/moa/core/Vote.java

```diff
@@ -0,0 +1,90 @@
+package org.apache.samoa.moa.core;
+
+/*
+ * #%L
+ * SAMOA
+ * %%
+ * Copyright (C) 2014 - 2015 Apache Software Foundation
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.samoa.moa.AbstractMOAObject;
+
+/**
+ * Class for storing votes.
+ * 
+ * @author Maciej Grzenda (maciej.grzenda@gmail.com)
+ * @author Jerzy Jegier (jerzyjeg@wp.pl)
+ *
+ */
+public class Vote extends AbstractMOAObject {
```

Comment:
"implements Serializable" instead of "extends AbstractMOAObject" should be enough.